### PR TITLE
Fix for parquet file schemes

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

File: dask/dataframe/io/parquet.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def to_parquet(path, df, compression=None, write_index=None, has_nulls=True,
 
     if append:
         pf = fastparquet.api.ParquetFile(path, open_with=myopen, sep=sep)
-        if pf.file_scheme != 'hive':
+        if pf.file_scheme not in ['hive', 'empty', 'flat']:
```
Comment (jakirkham) [inline dask/dataframe/io/parquet.py]: Is there a `"flat"` scheme somewhere? Am only finding `"empty"` and `"simple"`. Is it suppose to be `"simple"`?

Comment (martindurant) [reply dask/dataframe/io/parquet.py]: "flat" is new, when there are multiple files, but all are in one directory

**Comment by jakirkham** [comment]
The change seems clean and fine to me from a general perspective, but admittedly I don't know lots about parquet. Trusting your judgement and merging.

**Comment by jakirkham** [comment]
Just realized I forgot we merge squash. Would it make sense to just make this default (or only) kind of merge in the repo settings?

**Comment by martindurant** [comment]
There was only one commit, squash would not have made a difference

**Comment by mrocklin** [comment]
I usually squash merge by default.  I very rarely rebase-merge if the
commit history is particularly clean/concise/valuable.

**Comment by jcrist** [comment]
Just switched off merge-commits for this repo so this can't happen again.

**Comment by jakirkham** [comment]
Great! Thanks so much @jcrist. Sorry for the trouble.

**Pull request merged**